draft-ietf-httpbis-digest-headers.md

```diff
@@ -243,19 +245,21 @@ dependent on other transformations (e.g. transfer codings for HTTP/1.1 - see
 several examples are provided in {{resource-representation}}.
 
 When a message has no representation data it is still possible to assert that no
-representation data was sent computing the representation digest on an empty
+representation data was sent computing the digest on an empty
 string (see {{usage-in-signatures}}).
 
-`Representation-Digest` is a Structured Fields Dictionary (see {{Section 3.2 of
-STRUCTURED-FIELDS}}):
+`Representation-Digest` is a Structured Fields `Dictionary` (see {{Section 3.2 of
+STRUCTURED-FIELDS}}) where:
+* keys convey the hashing algorithm (see {{algorithms}})
+  used to compute the digest
+  and have no parameters;
```

Comment:
I don't follow this line, keys can't have parameters by design. If you mean that there are no parameters defined for Dictionary members, that probably needs to be done as a third bullet.

Reply:
I'll check the spec and fix